bentoml/_internal/server/grpc/server.py

```diff
@@ -43,7 +43,7 @@ def __init__(
         on_startup: t.Sequence[t.Callable[[], t.Any]] | None = None,
         on_shutdown: t.Sequence[t.Callable[[], t.Any]] | None = None,
         *,
-        _grace_period: int = 5,
+        _grace_period: int | None = None,
```

Comment:
set grace period sometimes leading to zombie process. set to `None` here and investigate later.